### PR TITLE
[history] keep track of user selections and order them at the top of the list

### PR DESCRIPTION
**Pull request by kyle-west**
We support History now.

### CODE REVIEW

File: build/content.js [hunk 1]
```diff
@@ -450,6 +475,40 @@ function sendMessage(command, options) {
...
+        const newHist = [...new Set([commandKey, ...hist])];
+        return new Promise(resolve => {
+            chrome.storage.local.set({ [userHistoryKey]: newHist }, () => {
+                resolve(newHist);
+            });
+        });
+    });
+}
+
+let commands$1 = commands;
```
Comment (zachintosh) [inline build/content.js:498]: What's the $ for?

**Review by zachintosh — APPROVED** [review]
Looks gooooooood

File: build/content.js (hunk 1 above)
Comment (kyle-west) [reply build/content.js:498]: I don't know. This is the built file so we don't manage it

Comment (zachintosh) [reply build/content.js:498]: 🤦  built file

**Pull request merged**